src/main/java/com/titarenko/service/ReaderImpl.java

```diff
@@ -0,0 +1,48 @@
+package com.titarenko.service;
+
+import com.titarenko.model.Employee;
+import com.titarenko.model.Gender;
+
+import java.util.*;
+
+public class ReaderImpl implements Reader {
+
```

Comment:
Name this class `ConsoleReaderImpl` and remove a `ReadFromConsole` class from this repo, please.